packages/components/app/styles/components/form/masked-input.scss

```diff
@@ -11,9 +11,13 @@ $hds-form-masked-input-button-size: 24px;
 
 .hds-form-masked-input {
   position: relative;
+  display: grid;
+  grid-template-areas: "input copy-button";
+  grid-template-columns: auto min-content;
```

Comment:
I am still unsure if this could (or should?) be `1fr auto` instead. But I don't have strong arguments for this gut feeling. 🤷‍♂️

Reply:
in this case, the outcome will be the same – will move to `1fr auto` as it seems to be more consistent with what we have elsewhere in the repo